package.json

```diff
@@ -66,7 +66,7 @@
   },
   "dependencies": {
     "@types/debug": "^4.1.12",
-    "@types/node": "~10.14.19",
+    "@types/node": "^22.13.9",
```

Comment:
wow, that's a sizeable version bump :joy:

Reply:
It sure is 😎 We've neglected updating that version